src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,179 @@ Keywords:
 
 # Release notes
 
+## March 20, 2024
+
+### iOS Mobile Core 5.x and compatible extensions
+
+New major version of the Mobile Core SDK for iOS has been released along with updates to other extensions. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 12.0 and tvOS 12.0 (for extensions which support tvOS).
+* Include XCFrameworks built with Xcode 15.0.1 with the GitHub release.
+* Include a privacy manifest for AEPCore and AEPServices.
+
+Note that all these extensions must be updated together.
+
+* iOS Mobile Core 5.0.0
+* iOS Signal 5.0.0
+* iOS Lifecycle 5.0.0
+* iOS Identity 5.0.0
+* iOS Profile 5.0.0
+* iOS Adobe Experience Platform Assurance 5.0.0
+* iOS Adobe Experience Platform Edge Network 5.0.0
+* iOS Identity for Edge Network 5.0.0
+* iOS Consent for Edge Network 5.0.0
+* iOS Media for Edge Network 5.0.0
+* iOS Edge Bridge 5.0.0
+* iOS Adobe Journey Optimizer 5.0.0
+* iOS Adobe Journey Optimizer - Decisioning 5.0.0
+* iOS Places Service 5.0.0
+* iOS Adobe Analytics 5.0.0
+* iOS Adobe Analytics - Media Analytics for Audio & Video 5.0.0
+* iOS Adobe Target 5.0.0
+* iOS Adobe Campaign Standard 5.0.0
+* iOS Adobe Campaign Classic 5.0.0
+* iOS Adobe Audience Manager 5.0.0
+
+### iOS Messaging 5.0.0
```

Comment:
nit: can we order these release notes as per the list above?

Reply:
Did you mean reorder the list based on the release notes?

Reply:
I prefer leaving it as they are currently grouped based on edge and solution extension.

Reply:
I mean reordering the release notes based on the list and grouping above.